src/config.hpp.in

```diff
@@ -21,7 +21,7 @@
 //  config.hpp file will be created (overwriting the last) from this template. This new
 //  file contains Athena++ specific cpp macros and definitions set by configure.
 
-#define COORDINATE_TYPE @COORDINATE_TYPE@
+#define COORDINATE_TYPE @COORDINATE_TYPE @
```

Comment:
wtf, i don't know how this happened.

Reply:
The formatter will do this if it runs on this file.